src/Controller/GameController.php

```diff
@@ -2,84 +2,97 @@
 
 namespace App\Controller;
 
-use App\Service\GameService;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\Routing\Annotation\Route;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\HttpFoundation\Cookie;
 use Symfony\Component\HttpFoundation\Request;
-use Symfony\Component\HttpFoundation\Session\SessionInterface;
+
+use App\Entity\Game;
+use App\Service\GameService;
 
 class GameController extends AbstractController
 {
     const COOKIE_KEY = 'pentago';
 
     private GameService $gameService;
-    private SessionInterface $session;
 
-    public function __construct(GameService $gameService, SessionInterface $session)
+    public function __construct(GameService $gameService)
     {
         $this->gameService = $gameService;
-        $this->session = $session;
     }
 
     /**
      * @Route("/", name="newGame")
      */
     public function newGame(): Response
     {
+        $entityManager = $this->getDoctrine()->getManager();
```

Comment:
You can keep your entity manager in controller :) Just for information when there is a lot of logic, it is better to move in a service, the controller must remain as small as possible. (easier to test, etc...)